src/__test__/comment.test.js

```diff
@@ -0,0 +1,27 @@
+/** * @jest-environment jsdom */
+
+import { commentCounter } from '../modules/comments';
+
+document.body.innerHTML = `
+        <ul class="allComments">
+        <li>2022-04-21 Mohammed Elseddik: good recipe</li>
+        <li>2022-04-21 hello : amazign recipe</li>
+        <li>2022-04-21 hello : I loved thsi recipe</li>
+        </ul>
+`;
+
+const data = [{ comment: 'amazing', userName: 'Evans', date: '12/12/2112' }, { comment: 'i LOVE IT', userName: 'CRIS', data: '22/02/2020' }];
+const commentItem = document.querySelector('.allComments');
+data.forEach((elem) => {
+  const newComment = document.createElement('li');
+  newComment.innerHTML += `${elem.creation_date} ${elem.username}: ${elem.comment}`;
+  commentItem.appendChild(newComment);
+});
+console.log([commentItem.children].length);
+
```

Comment:
### OPTIONAL

_**I write it as an optional but this is a requirement that is not mentioned in the last review. So please fix it.**_

- [ ] You should delete the console.log as last TSE said. Last TSE said optional but this is a [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md#:~:text=Do%20not%20commit%20console.log%20to%20your%20repo.%20It%E2%80%99s%20ugly%2C%20it%20kills%20performance%20and%20it%20can%20make%20confidential%20data%20be%20visible%20to%20anyone%20using%20the%20browser%20tools%20to%20look%20at%20your%20website.).